bin/update_or_create_webhooks.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+
+import click
+from openedx_webhooks.lib.github.utils import update_or_create_webhook
+
+click.disable_unicode_literals_warning = True
+
+
+@click.command()
+@click.argument('config-json', type=click.File('rb'))
+def cli(config_json):
+    """
+    Update or create the specified webhook into repos using a
+    configuration JSON file:
+
+    \b
```

Comment:
Bleh: http://click.pocoo.org/5/documentation/#preventing-rewrapping   

Click couldn't have used some existing .rst syntax? Ugh. Or if they're going to make up something of their own, use something readable instead of a backspace character.... Carry on.